plugins/services/src/js/components/forms/GeneralServiceFormSection.js

```diff
@@ -274,7 +276,10 @@ class GeneralServiceFormSection extends Component {
             </FieldError>
           </FormGroup>
 
-          <FormGroup className="flex flex-item-align-end column-2 flush-left">
+          <FormGroup className={{
```

Comment:
It's rather confusing actually. Because it is not a native behavior, but `FormGroup` component that uses `classNames` library underneath. I'm not convinced it is a good idea to keep it like that.